session/src/test/java/org/apache/iotdb/session/SessionUT.java

```diff
@@ -0,0 +1,81 @@
+package org.apache.iotdb.session;
+
+import org.apache.iotdb.rpc.BatchExecutionException;
+import org.apache.iotdb.rpc.IoTDBConnectionException;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+import org.apache.iotdb.tsfile.write.record.Tablet;
+import org.apache.iotdb.tsfile.write.schema.MeasurementSchema;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SessionUT {
+
+    public static void main(String[] args) throws BatchExecutionException, IoTDBConnectionException {
```

Comment:
Thanks! We usually use JUnit framework in test, not main thread. You could refer to org.apache.iotdb.tsfile.read.common.BatchDataTest to see how to Assert